docs/documentation/server_admin/topics/authentication/passkeys.adoc

```diff
@@ -12,3 +12,80 @@ Therefore, users of {project_name} can do Passkey registration and authenticatio
+
+. Configure the `WebAuthn Passwordless Policy`. Perform the configuration in the Admin Console, `Authentication` section,  in the tab `Policies` ->  `WebAuthn Passwordless Policy`. Set *User Verification Requirement* to *required* and *Require discoverable credential* to *Yes* when you configure the policy for loginless scenario. Note that since there is no dedicated Loginless policy, it is impossible to mix authentication scenarios with user verification=no/discoverable credential=no and loginless scenarios (user verification=yes/discoverable credential=yes).
++
+NOTE: Storage capacity is usually very limited on hardware passkeys meaning that you cannot store many discoverable credentials on your passkey. However this limitation may be mitigated for instance if you use Android phone backed by Google account as a passkey device or iPhone backed by Bitwarden.
+
+. Configure the authentication flow. Create a new authentication flow, add the *Passkeys Conditional UI Authenticator* execution and set the Requirement setting of the execution to *Required*
++
+The final configuration of the flow looks similar to this:
+image:images/passkey-conditional-ui-flow.png[Passkey Authentication with Conditional UI flow flow]
+
+. Bind the flow above as *browser* authentication flow in the realm as described in the <<_webauthn-register, WebAuthn section above>>.
+
+The authentication flow above requires that user must already have passkey credential on his account to be able to login. This means that all users in the realm must have passkeys already set.
+That can be achieved for instance by enable user registration as described below.
+
+===== Setup of the registration for passkeys conditional UI
+
+. Enable <<con-user-registration_{context}, registration>> for your realm
+
+. In the <<configuring-authentication_{context},authentication flows>> of the realm, select flow *registration* and switch the authenticator *Password validation* to *Disabled*.
+This means that newly registered users will not be required to create the passwords as for this example setup, we want the users to always use passkeys instead of the passwords.
+
+. Go back to *Required actions* sub-tab of the tab *Authentication* tab and find the `Webauthn Register Passwordless` action and mark it with *Set as default action*.
+This means that it would be added to all new users after their registration.
+
+The alternative to the registration flow setup is to add the required action `WebAuthn Register Passwordless` to a user who is already known to {project_name}, to test this. The user with the required action configured will have to authenticate (with a username/password for example) and will then be prompted to register a passkey to be used for loginless authentication.
```

Comment:
Not sure what "to test this" means.

Reply:
"to test this" is not needed so that I removed the words. Thank you.